exporter/otelarrowexporter/metadata.yaml

```diff
@@ -0,0 +1,9 @@
+type: otelarrow
+
+status:
+  class: exporter
+  stability:
+    development: [traces, metrics, logs]
+  distributions: [contrib]
+  codeowners:
+    active: [jmacd, moh-osman3]
```

Comment:
this doesn't match the code owners listed here: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/30619/files#diff-3d36a1bf06148bc6ba1ce2ed3d19de32ea708d955fed212c0d27c536f0bd4da7R71

Reply:
Fixed.